packages/client/e2e/limit-checker.spec.ts

```diff
@@ -50,26 +52,35 @@ test.describe("Limit Checker", () => {
 
     await tilePage.locator("[data-testid='Buy-EURUSD']").click()
 
-    await tradeBlotterFirstRowCells.nth(1).getByText(lastTradeId + 1 + "").waitFor({state: "visible", timeout:5000})
-    
-    await assertGridRow(
-      limitTableFirstRowCells,
-      [limitId + 1 + "", "Success", "EURUSD", "2,000,000"],
-      1,
-      5,
-    )
-
-    await assertGridRow(
-      tradeBlotterFirstRowCells,
-      ["Buy", "EURUSD", "EUR", "1,999,999"],
-      4,
-      8,
-    )
+    await tradeBlotterFirstRowCells
+      .nth(1)
+      .getByText(lastTradeId + 1 + "")
+      .waitFor({ state: "visible", timeout: 5000 })
+
+    await assertGridRow({
+      row: limitTableFirstRowCells,
+      assertions: [
+        limitId + 1 + "",
+        "Success",
+        "EURUSD",
+        "1,999,999",
+        "2,000,000",
+      ],
+      firstCellToAssert: 1,
+      lastCellToAssert: 6,
+    })
+
+    await assertGridRow({
+      row: tradeBlotterFirstRowCells,
+      assertions: ["Buy", "EURUSD", "EUR", "1,999,999"],
+      firstCellToAssert: 4,
+      lastCellToAssert: 8,
```

Comment:
little bit confusing that this means the previous cell is the last one to actually assert on .. right?